lucene/core/src/java/org/apache/lucene/index/IndexWriterConfig.java

```diff
@@ -110,7 +110,7 @@
   /** Default value for compound file system for newly written segments
    *  (set to <code>true</code>). For batch indexing with very large 
    *  ram buffers use <code>false</code> */
-  public final static boolean DEFAULT_USE_COMPOUND_FILE_SYSTEM = true;
+  public final static boolean DEFAULT_USE_COMPOUND_FILE_SYSTEM = false;
```

Comment:
same as below

Reply:
this thing also for remove compound file formation so we can see our file sizes more accurately